pkg/datapath/iptables/iptables.go

```diff
@@ -28,10 +34,6 @@ import (
 	"github.com/cilium/cilium/pkg/option"
 	"github.com/cilium/cilium/pkg/sysctl"
 	"github.com/cilium/cilium/pkg/versioncheck"
-
-	"github.com/blang/semver/v4"
-	"github.com/mattn/go-shellwords"
-	"github.com/sirupsen/logrus"
```

Comment:
You mean you used the master linter to check that PR instead of v1.11's? Anyway, not worth repushing just for that IMO.

Reply:
> You mean you used the master linter to check that PR instead of v1.11's?

No, those changes were in master because of the linter, and I ended up pulling them here

> Anyway, not worth repushing just for that IMO.

:+1:

Reply:
> No, those changes were in master because of the linter, and I ended up pulling them here

That's shouldn't happen given they are not part of the backported commits and well separated from other changes. That's why I'm confused. Anyway, nit.